src/providers/authenticationProvider.ts

```diff
@@ -280,28 +280,28 @@ export class TerraformCloudAuthenticationProvider implements vscode.Authenticati
     const terraformCloudURL = `${TerraformCloudWebUrl}/settings/tokens?source=vscode-terraform`;
     let token: string | undefined;
     switch (choice.label) {
-      case 'Open to generate a User token':
+      case 'Generate a user token':
```

Comment:
As far as I remember, this choice cannot be accompanied with an icon to indicate it would open a browser (new window), so I think the more verbose version of the label is still helpful here. I'd perhaps even make it more verbose - e.g. `Open browser to generate a token` or `Open ${TerraformCloudWebUrl} to generate a token` (or just a hostname) or something similar.

Reply:
The `detail` part right under this line says `Open the Terraform Cloud website to generate a new token`, which I thought was explanation enough to shorten the label alongside the guidelines in the ticket

I could add a globe, but then the others look like something is missing or not rendered:

<img width="725" alt="image" src="https://github.com/hashicorp/vscode-terraform/assets/272569/3fb06886-9621-4d61-b420-143137c82199">

I could add other icons, but then we'd have to discuss which to use:

<img width="726" alt="image" src="https://github.com/hashicorp/vscode-terraform/assets/272569/21889f87-afe7-4cae-a7a4-174dfcb6a24a">

Reply:
I completely missed the fact there's the other more detailed explanation right below! Apologies.